test/plugin/scenarios/postgresql-above9.4.1207-scenario/config/expectedData.yaml

```diff
@@ -53,6 +53,7 @@ segmentItems:
         - {key: "db.type", value: "sql"}
         - {key: "db.instance", value: "postgres"}
         - {key: "db.statement", value: "INSERT INTO test_007(id, value) VALUES(?,?)"}
+        - {key: "db.bind_vars", value: "[1,1]"}
```

Comment:
And after setting the default value of `TRACE_SQL_PARAMETERS` to `false`, we should pass an environment variable to override it to `true`, otherwise, there is no key `db.sql.parameters`

Reply:
how to set?

Reply:
in docker setting?

Reply:
seems to be here

https://github.com/apache/skywalking/blob/2e2616d0efc34c29a3d3388e100d948303ad2832/test/plugin/scenarios/postgresql-scenario/bin/startup.sh#L20

you can use something like 

```
java -jar ${agent_opts} -Dskywalking.plugin.postgresql.trace_sql_parameters=true ${home}/../libs/posgresql.......
```

it should work, please try

Reply:
ok